CHANGELOG.md

```diff
@@ -724,6 +730,7 @@ the code was deployed.
 - Initial pi.
 
 [unreleased]: https://github.com/bravetechnologycoop/BraveButtons/compare/v13.8.0...HEAD
+[13.9.0]: https://github.com/bravetechnologycoop/BraveButtons/compare/v13.7.1...v13.9.0
```

Comment:
Similarly, this does not need to be changed until we deploy to production

Reply:
Fixed this, I left it in the same format as above. Let me know if that isn't ok.